qsdsan/sanunits/_sludge_treatment.py

```diff
@@ -239,7 +239,7 @@ def _cal_thickener_factor(self, TSS_in):
             if thickener_factor<1:
                 thickener_factor=1
             return thickener_factor
-        else: return None
+        else: raise ValueError(f'Influent TSS is not valid ({TSS_in:.2f} mg/L).')
```

Comment:
@RaiSaumitra @joyxyz1994 previously, it returns `None` when `TSS_in` <=0, but then the error is vague (i.e., the users won't be able to readily tell what is wrong), so I updated the error message, does this make sense or are there situations where `None` should be returned?

Reply:
Looks good to me!